setup.py

```diff
@@ -49,7 +49,7 @@ def _read_reqs(relpath):
 
 setuptools.setup(
     name="biggraphite",
-    version="0.14.6",
+    version="0.14.7",
```

Comment:
Nitpicking: I would prefer to do version bumps in separate pull-requests.

Otherwise, you may append fix commits to the PR (after the version bump) and it would be misleading to tag the release.

Reply:
Ack, will rebase before merging to put it on top